model/src/main/proto/javaclasses/exlibris/q/projections.proto

```diff
@@ -87,3 +89,252 @@ message HomePage {
     repeated Category category = 2;
 }
 // *** This is projections for UI role librarian ***
+// View for adding book.
+//
+// View consists of fields for writing book details.
+//
+message AddBookView {
+
+    //List of book categories.
+    repeated Category categories = 1;
+}
+
+// View for editing book.
+//
+// View consists of fields for edit book details.
+//
+message UpdateBookView {
+
+    // URL for book cover.
+    spine.net.Url url = 1;
+
+    // Value that generated from decimal ISBN.
+    Isbn62 isbn = 2;
+
+    // Book’s title.
+    BookTitle title = 3;
+
+    // Full author’s name.
+    repeated Author authors = 4;
+
+    // Year of publication of the book.
+    Year year = 5;
+
+    // Categories of book.
+    repeated Category categories = 6;
+
+    // Full book’s description.
+    BookDescription description = 7;
+}
+
+// View for appending inventory.
+//
+// The second step of wizard of adding book items.
+//
+message AppendInventoryView {
+
+    // An identifier is represented by BookId.
+    InventoryId inventory_id = 2;
+}
+
+// View with compact information about book’s loans.
+//
+// Librarian sees all users that borrow book and the loan’s period.
+//
+message BookCompactHistory {
+
+    // URL for book cover.
+    spine.net.Url url = 1;
+
+    // List of user's loan details.
+    repeated LoanDetails loans = 2;
+}
+
+// View with information about book’s loan.
+//
+// View consists of fields about inventory item,
+// users who borrowed the book and their taken on and due on dates.
+//
+message LoanDetails {
+
+    // An identifier of a book. Contains ISBN base62.
```

Comment:
The identifier of item contains not only ISBN with base62, it is ISBN and number that inputted by librarian too.